types/cesium/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for cesium 1.66
+// Type definitions for cesium 1.67
```

Comment:
It doesn't look like this PR covers all of the additions/removals in 1.67 [1], so I don't think we should bump this yet.

1: https://github.com/CesiumGS/cesium/blob/master/CHANGES.md#1670---2020-03-02

Reply:
Reverted that change, thanks for the review!